### ISSUE TITLE
Clean up isDeprecated values

### ISSUE THREAD

**Issue by swinslow** [spdx/license-list-XML#1545]
(currently marking for 3.19 docs/cleanup release, but might be more appropriate for 3.20 instead)

This is a follow-on from the discussion in #583. I believe @goneall has reworked the license list publisher tooling to just look for the presence of `deprecatedVersion="..."` to determine whether or not a license is deprecated. Which means that the separate `isDeprecated=true` flag is (ironically) deprecated.

Two tasks to do in parallel:
* remove `isDeprecated` from the [XML schema description file](https://github.com/spdx/license-list-XML/blob/master/schema/ListedLicense.xsd); and
* go through the deprecated licenses and exceptions to make sure that they have a `deprecatedVersion="..."` value and to remove the `isDeprecated=true` flag.

**Comment by swinslow** [comment]
Moving to 3.21, this is not urgent for 3.20.

Would encourage folks who are new-ish to contributing to the license list to consider helping with this one (since it is good for new contributors, like the tag says). Raise your hand and I'm happy to help advise on what is needed here  ;-)

**Comment by BassCoder2808** [comment]
Hi @swinslow, if the issue is still open, I am happy to contribute to this. Can you please tell me like how should I proceed forward to solve this issue

**Comment by swinslow** [comment]
Hello @BassCoder2808,

Yes, this issue is still open. Feel free to review the details above and help would be welcome!

You'll probably want to start by reviewing the comments in this thread and the two bullet points noted in the intro. The second bullet point is the one you'll want to start with, by going through the licenses in the [`src/` folder](https://github.com/spdx/license-list-XML/tree/main/src) and exceptions in the [`src/exceptions/` folder](https://github.com/spdx/license-list-XML/tree/main/src/exceptions).

Identify which ones, if any, have a `<license>` or `<exception>` tag **with** an `isDeprecated=` attribute but **without** a `deprecatedVersion=` value.

Once you've identified the licenses and exceptions that meet those criteria, you'll likely want to paste the list of identifiers into a comment here, so that the SPDX legal community can review and help identify which license list release versions correspond to when those identifiers were deprecated.

Then you could use that information to insert the missing `deprecatedVersion=` values and remove the now-unnecessary `isDeprecated=` fields. You'd want to submit a PR making those changes, and we can confirm it passes the CI checks and then merge it.

And finally, after that is complete and merged, you could submit a second PR to remove `isDeprecated=` from the [XML schema description file](https://github.com/spdx/license-list-XML/blob/master/schema/ListedLicense.xsd).

Hope this helps!

**Comment by BassCoder2808** [comment]
Thanks a lot @swinslow for your guidance, now I have got a better idea about how to proceed further, will go through the steps and will let you know if I face any difficulties

**Comment by BassCoder2808** [comment]
@swinslow I think I have completed the first task, like I have found 34 licenses all in the `src` folder which contain the `isDeprecated` field, they have the corresponding `deprecatedVersion` associated with it in the XML file. So should I proceed further in solving the second part of removing the `isDeprecated` part from these files

**Comment by swinslow** [comment]
Thank you @BassCoder2808! Just to confirm, are there any licenses or exceptions where `isDeprecated` is present but `deprecatedVersion` is **not** present?

**Comment by BassCoder2808** [comment]
No, there are no licenses where `isDeprecated` is present and the value of `deprecatedVersion` is not set

**Comment by BassCoder2808** [comment]
Hi @swinslow , as the first part has been merged, should I remove the `isDeprecated` attribute  from the `XML Schema Definition` file. Also, I just saw that the documentation file `xml-fields.md` contains `isDeprecated` which I think also needs to be removed now, so should I make these 2 changes through a separate issue or through a single PR representing this issue itself

**Comment by swinslow** [comment]
(re-opening as the schema still needs to be updated)

Hi @BassCoder2808, yes, thank you! You're correct, we'll also want to remove references to it in the documentation. If you can submit a single PR that makes both of those changes (and closes this issue as fixed), that would be perfect. Thank you!

**Comment by BassCoder2808** [comment]
Hi @swinslow , I have created a new pull request for the same, please review once, and let me know if anything else needs to be changed
